library/alloc/src/collections/btree/map.rs

```diff
@@ -149,40 +149,34 @@ impl<K: Clone, V: Clone> Clone for BTreeMap<K, V> {
     fn clone(&self) -> BTreeMap<K, V> {
         fn clone_subtree<'a, K: Clone, V: Clone>(
```

Comment:
Previously support functions have move to support modules to downsize map.rs, but I don't know where to move this.